### PR TITLE
v0.1.0 (part 1)

### LINKED ISSUE #23: Separate `main.rs`
`main.rs` is getting out of hand length-wise, so we need to separate it into different files.

### LINKED ISSUE #26: BUG ・PathBuf not working on MacOS
### Before you start, first check the following:

- [X] I have searched the issues and confirmed that this is not a duplicate or a known issue.
- [X] I am on the newest version of Notabena.

### Are you facing this issue using Notabena CLI or Desktop?

CLI

### What platform are you using Notabena on?

macOS

### What kind of bug are you facing?

Hi,
So i tried running notabena by ``` cargo run ```, but it gave me this error:
    Finished dev [unoptimized + debuginfo] target(s) in 0.13s
     Running `target/debug/notabena`
PathBuf { inner: "/Users/dmarhitych-pro/Library/Notabena" }
Error: Os { code: 2, kind: NotFound, message: "No such file or directory" }
I don't understand why.
Can you help me?

### How to reproduce the bug?

1. Cargo run on macos

### What did you expect that would happen?

To just run

### Can you send us some screenshots?

Screenshot 2023-07-18 at 23 44 10

### Anything to add?

_No response_

### PR DESCRIPTION
**Pull request by MrSerge01**
# major changes
- separated main.rs into multiple smaller files (#23)
- fixed bug of database storage (#26)

### CODE REVIEW

**Comment by MrSerge01** [comment]
cancelled making it a mega pull request that solves the world

File: src/database/edit.rs
Comment (SwiftDust) [inline src/database/edit.rs:1]: Empty file? What's it supposed to do?

File: src/main.rs [hunk 1]
```diff
@@ -1,19 +1,20 @@
 // TODO: Organize the code into different files
```
Comment (SwiftDust) [inline src/main.rs]:
```suggestion

```

This line can be removed for now.

File: src/database/show.rs
Comment (SwiftDust) [inline src/database/show.rs:1]: Empty file? What's it supposed to do?

File: src/output.rs
Comment (SwiftDust) [inline src/output.rs:1]: This file does entirely nothing. Can it be deleted?

**Review by SwiftDust — CHANGES_REQUESTED** [review]
Please provide clarity by adding comments or remove the files.

**Comment by SwiftDust** [comment]
Clarity has been provided by @MrSerge01 through discord.

**Pull request merged**